<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SymmGaitModelsClass</class>
 <widget class="QWidget" name="SymmGaitModelsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SymmGaitModels</string>
  </property>
  <widget class="QPushButton" name="RepaintButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Repaint</string>
   </property>
  </widget>
  <widget class="GLWidget" name="OpenGLWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>560</width>
     <height>460</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="MarkerNrLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="MarkerLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Marker Number (1-26)</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="CoordRadioGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Coordinate</string>
   </property>
   <widget class="QRadioButton" name="XRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="YRadioButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ZRadioButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="LoadMatrixButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Another Matrix</string>
   </property>
  </widget>
  <widget class="QLabel" name="LoadedMatrixTitleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Loaded Matrix:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LoadedMatrixNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>glwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="symmgaitmodels.qrc"/>
 </resources>
 <connections/>
</ui>
